src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
```

Comment:
Each variable must be responsible for what is contained within it.
Use the more clear name.
For example:

```suggestion
  const stylesObject = {};
```